server/kit/kitserver.go

```diff
@@ -103,7 +103,16 @@ func NewServer(svc Service) *Server {
 				"message", "unable to initiate error reporting client")
 		}
 
-		err = initSDExporter(projectID, serviceID, svcVersion, lg)
+		err = initSDExporter(gaeSDExporterOptions(projectID, serviceID, svcVersion, lg))
+		if err != nil {
+			lg.Log("error", err,
+				"message", "unable to initiate error tracing exporter")
+		}
+
+		propr = &sdpropagation.HTTPFormat{}
+	} else if isGKE() {
+		projectID := googleProjectID()
+		err = initSDExporter(gkeSDExporterOptions(projectID, lg))
```

Comment:
I notice you didn't bring the error reporting client over as well for GKE/others. I think it should still work outside of GAE so probably trying to hook in here as well?

Reply:
yeah. adding it